regolith/app.py

```diff
@@ -59,6 +61,17 @@ def collection_page(dbname, collname):
                     f.write(form['body'])
                 traceback.print_exc()
                 raise
+            tv, errors = validate(dbname, body)
+            if not tv:
+                td = tempfile.TemporaryDirectory()
+                n = os.path.join(td.name, 'regolith.txt')
+                print(errors)
+                print('Writing text file to {}. '
+                      'Please try again.'.format(n))
```

Comment:
I think these prints should go into the body of the error message below. At the very least, they should be printed to stderr....